libraries/fabric-shim/test/unit/server.js

```diff
@@ -76,9 +76,6 @@ describe('ChaincodeServer', () => {
         it('should create a gRPC server instance and call addService in the constructor', () => {
```

Comment:
This test no longer checks if "addService in the constructor" is called. Not sure what it does test really.